integrasjon/dokgen-klient/src/main/java/no/nav/foreldrepenger/melding/integrasjon/dokgen/dto/opphørSVP/OpphørPeriode.java

```diff
@@ -0,0 +1,100 @@
+package no.nav.foreldrepenger.melding.integrasjon.dokgen.dto.opphørSVP;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import no.nav.foreldrepenger.melding.geografisk.Språkkode;
+import no.nav.foreldrepenger.melding.integrasjon.dokgen.dto.felles.Årsak;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Objects;
+
+import static no.nav.foreldrepenger.melding.typer.Dato.formaterDato;
+
+public class OpphørPeriode {
+    private Årsak årsak;
+    @JsonIgnore
+    private LocalDate periodeFom;
+    private String stønadsperiodeFom;
```

Comment:
Synes LocalDate-versjonen bør hete `stønadsperiodeFomDate` for å synliggjøre at det er samme verdien i annet format, slik vi har gjort det på alle de andre DTOene.

Reply:
utført